enterprise_access/apps/api_client/lms_client.py

```diff
@@ -156,9 +179,9 @@ def fetch_group_members(
             - ``page`` (int, optional): Which page of paginated data to return. Cannot be supplied if
             ``traverse_pagination`` is supplied.
         """
-        if traverse_pagination and page:
+        if traverse_pagination == page:
```

Comment:
This check only works if page == 1.  Numbers above 1 aren't considered equal to True.